CriteoPublisherSdk/Sources/Configuration/SkanInfo.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SkanInfo.swift
+//  CriteoPublisherSdk
+//
+//  Copyright © 2018-2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+struct SkanInfo {
```

Comment:
Out of context, Skan is not expressive and hard to understand (like the ttnc stuff you removed earlier).
What do you think about renaming this to a longer form ?

Reply:
We already use the `Skan` acronym everywhere, (unlike the ttnc).

Also I was reluctant to name this following the Apple naming prefix `SKAdNetwork` as it could be confusing or worse name clashing. But since this will be in a different module, that shouldn't be an issue.
Let's even push further that and try a `SKAdNetwork` extension 😂.

Reply:
Done in this [push](https://github.com/criteo/ios-publisher-sdk/compare/6b1aaa4b6601c57cb146406b0ac05b16334c3466..73d60fa392d6fcf899c732d3d280dc5122db7e7c)